### DISCUSSION TITLE
Пример дашборда

### DISCUSSION THREAD

**Discussion by Barrya42** [LazarenkoA/prometheus_1C_exporter#46] [Q&A] [answered]
Было бы отлично, если бы вы приложили пример файла дашборда, с котороро сделаны скрины. Будет намного проще внедрять вашу работу. Это поможет многим пользователям, втч и мне

**Comment by LazarenkoA** [comment, marked as answer]
в ишью выкладывали https://github.com/LazarenkoA/prometheus_1C_exporter/issues/26

**Reply by Barrya42** [reply]
Спасибо. Не увидел закрытые ишью.
